images/virtualization-artifact/pkg/controller/vm/vm_controller.go

```diff
@@ -44,29 +45,30 @@ func NewController(
 	log *slog.Logger,
 	dvcrSettings *dvcr.Settings,
 ) (controller.Controller, error) {
-	if log == nil {
```

Comment:
Please, tell me why this check is no longer relevant?

Reply:
I found this check redundant, there is no correct case when logger is nil